ui/src/app/components/elements/list.component.ts

```diff
@@ -518,4 +518,23 @@ export class ElementsListComponent extends BaseComponent implements OnInit {
   humanizedDate(date) {
     return moment.duration(moment().diff(date)).humanize() + ' ago';
   }
+
+  search(term?: string) {
+    if (term) {
+      if (!this.query?.includes('workspaceVersionId')) {
+        this.query += ",workspaceVersionId:" + this.versionId;
+      }
+      if (this.query?.includes('name')) {
```

Comment:
name string will always be there, on subsequent query search, right?

Reply:
yes bro